src/Database/Query/SelectQuery.php

```diff
@@ -232,6 +234,40 @@ public function distinct(ExpressionInterface|array|string|bool $on = [], bool $o
         return $this;
     }
 
+    /**
+     * Add engine-specific optimizer hint.
+     *
+     * @param string $name Hint name
+     * @param array|string $parameters Hint parameters
+     * @param bool $overwrite Whether to replace existing hints
+     * @return $this
+     */
+    public function hint(string $name, array|string $parameters, bool $overwrite = false)
```

Comment:
Can you give an example of a usage call and the generated hint?

Reply:
Will quoting of the params be required/supported?

Reply:
Updated parameters.